install/traf_checkset_env.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
```

Comment:
This file needs an Apache license header. (This is why the static checks are failing.)